draft-ietf-httpbis-http2bis.xml

```diff
@@ -1483,7 +1483,7 @@ DATA Frame {
 
   [Pad Length (8)],
   Data (..),
-  Padding (..),
+  Padding (..255),
```

Comment:
With only one byte for pad length, the length is in effect capped here.
This is a separate commit for ease of reverting, if needed.

Reply:
No, this is a good catch, thanks.

Reply:
Mostly...

```suggestion
  Padding (..2040),
```

Reply:
Oops, good catch.  I pushed a fixup for the other two instances.